tests/test_dataset.py

```diff
@@ -603,6 +603,40 @@ def test_is_isomorphous(data_unmerged, data_fmodel, sg1, sg2, cell1, cell2):
             assert not result
 
 
+@pytest.mark.parametrize("threshold", [5.0, 1.0, 0.5, 0.1])
+@pytest.mark.parametrize("isomorphous", [True, False])
+@pytest.mark.parametrize("sign", [True, False])  # Which cell is bigger
```

Comment:
`sign` is a bit confusing, I anticipated something being positive or negative.

What about `@pytest.mark.parametrize("compare_to", ["bigger_cell", "smaller_cell"])`, then compare strings in what follows?

Reply:
Okay I clarified it.